config/cmake/modules/FindRAJA.cmake

```diff
@@ -14,17 +14,35 @@
 #   - RAJA_LIBRARIES
 #   - RAJA_INCLUDE_DIRS
 
-include(MfemCmakeUtilities)
-mfem_find_package(RAJA RAJA RAJA_DIR "include" "RAJA/RAJA.hpp" "lib" "RAJA"
-  "Paths to headers required by RAJA." "Libraries required by RAJA.")
-
-if (NOT RAJA_CONFIG_CMAKE)
-   set(RAJA_CONFIG_CMAKE "${RAJA_DIR}/share/raja/cmake/raja-config.cmake")
+if (RAJA_FOUND)
+   return()
+endif()
+message(STATUS "Looking for RAJA ...")
+if (RAJA_DIR)
+   message(STATUS "   in RAJA_DIR = ${RAJA_DIR}")
+   # As a workaround for an issue in raja-config.cmake we try to load
+   # bltTargets.cmake before calling find_package(RAJA ...).
+   if (NOT RAJA_BLT_TARGETS)
+      set(RAJA_BLT_TARGETS "${RAJA_DIR}/lib/cmake/raja/bltTargets.cmake")
+   endif()
+   if (EXISTS "${RAJA_BLT_TARGETS}")
+      include("${RAJA_BLT_TARGETS}")
+   endif()
```

Comment:
With the new 2022.10.3 release of RAJA we can probably remove this workaround but we should test without it to make sure it is no longer necessary.

Reply:
Note: when testing with HIP, I had to use the fix from #3347.